src/main/java/frc/robot/commands/ShootNoteCommand.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.ScoringConstants;
+import frc.robot.subsystems.Scoring;
+
+public class ShootNoteCommand extends Command {
+    private Scoring scoring = Scoring.getInstance();
+    private Timer timer = new Timer();
+    private int secondsToScore;
+
+    public ShootNoteCommand(int secondsToScore) {
+        this.addRequirements(scoring);
+        this.secondsToScore = secondsToScore;
+    }
+
+    public ShootNoteCommand() {
+        this.addRequirements(scoring);
+    }
+
+    @Override
+    public void initialize() {
+        timer.start();
+    }
+
+    @Override
+    public void execute() {
+      if (scoring.getPosition() != ScoringConstants.groundSetpoint) {
+          scoring.rollerShoot();
+      }
+
+      // can i run the position command inside this command
```

Comment:
Is this comment indicating something is not yet complete? if so it should look like `// TODO: whatever part is intentionally not completed`